File: scripts/extract_names.py
import pandas as pd

df = pd.read_csv('/vercel/share/v0-project/data/processed_characters.csv')
names = df['name'].tolist()

print(f"Total characters: {len(names)}")
print("---")
for i, name in enumerate(names, 1):
    print(f"{i}. {name}")

File: scripts/extract_names.mjs
import { readFileSync } from 'fs';
import { parse } from 'csv-parse/sync';

const csvContent = readFileSync('./data/processed_characters.csv', 'utf-8');
const records = parse(csvContent, {
  columns: true,
  skip_empty_lines: true,
  relax_column_count: true,
  relax_quotes: true,
});

console.log(`Total characters: ${records.length}`);
console.log('---');

const names = records.map(r => r.name).filter(Boolean);
names.forEach((name, i) => {
  console.log(`${i + 1}. ${name}`);
});
